php-classes/Emergence/Connectors/Exceptions/SyncException.php

```diff
@@ -29,18 +29,13 @@ public function getContext($key = null)
         return $this->context;
     }
 
-    protected static function interpolate($message, array $context = array())
+    public static function interpolate($message, array $context = [])
```

Comment:
@nbey I think we'd be fine to put the actual interpolate code as a static method in `Emergence\Logger`